package.json

```diff
@@ -4,8 +4,10 @@
   "repository": "https://github.com/getfider/fider",
   "private": true,
   "dependencies": {
+    "@lingui/core": "^3.13.3",
```

Comment:
Why did you need to change these dependencies here?

Reply:
I'm not developer, so I used Cursor to fix Arabic issue